camel/toolkits/hybrid_browser_toolkit/ts/websocket-server.js

```
@@ -0,0 +1,210 @@
+      case 'get_page_snapshot':
+        if (!this.toolkit) throw new Error('Toolkit not initialized');
+        return await this.toolkit.getPageSnapshot(params.viewport_limit);
+
+      case 'get_snapshot_for_ai':
+        if (!this.toolkit) throw new Error('Toolkit not initialized');
+        return await this.toolkit.getSnapshotForAI();
+
+      case 'get_som_screenshot':
+        if (!this.toolkit) throw new Error('Toolkit not initialized');
+        console.log('Starting screenshot...');
+        const startTime = Date.now();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)